src/main/java/org/kiwiproject/test/junit/jupiter/MongoServerExtension.java

```diff
@@ -102,11 +106,17 @@
     @Getter
     private final DropTime dropTime;
 
+    /**
+     * The version of the mongo server to use;
+     */
+    @Getter
+    private final ServerVersion serverVersion;
```

Comment:
The `serverVersion` field is never actually used anywhere. I am guessing you meant to pass it as the argument to `startInMemoryMongoServer` in the `beforeAll` like this:

```java
mongoServer = startInMemoryMongoServer(serverVersion);
```

Reply:
You are correct.